packages/devextreme-vue/src/core/component.ts

```diff
@@ -331,6 +340,8 @@ function initDxComponent() {
       thisComponent.$_instance.endUpdate();
 
       restoreNodes(this.$el, nodes);
+      pickOutDxClasses(this.$el, this.$attrs);
```

Comment:
I'm also worried that if we pick classes on mount, will they be relevant at the moment we restore them?

I mean, the Widget may change its dx classes dynamically, are we sure that what we pick here stays relevant at the moment of restoration?